app/models/user.rb

```diff
@@ -1,4 +1,4 @@
-# typed: strict
```

Comment:
@mjgiarlo I think `bundle exec srb rbi suggest-typed` did this.  Do we want this?

Reply:
@jcoyne It probably thinks it's helping us. To test that, you can try flipping that back to `typed: strict` and running `srb tc` to see if anything breaks?

Reply:
Also, it seems weird that sorbet-rails removed so many definitions. but whatever?